s3tests_boto3/functional/test_s3select.py

```diff
@@ -317,6 +376,17 @@ def test_count_operation():
 
     s3select_assert_result( num_of_rows, int( res ))
 
+@attr('s3select')
```

Comment:
for pytest, you'll need to replace `@attr('s3select')` with the new `@pytest.mark.s3select`